Makefile

```diff
@@ -76,12 +76,18 @@ EXTERNS_ANGULAR_HTTP_PROMISE = .build/externs/angular-1.4-http-promise_templated
 EXTERNS_JQUERY = .build/externs/jquery-1.9.js
 EXTERNS_FILES = $(EXTERNS_ANGULAR) $(EXTERNS_ANGULAR_Q) $(EXTERNS_ANGULAR_HTTP_PROMISE) $(EXTERNS_JQUERY)
 
+OS := $(shell uname)
+
```

Comment:
For me all this OS specific block should be at the top of the file (actually we use the variable before we set them, it's working but it's not natural ...)